lnrpc/walletrpc/walletkit_server.go

```diff
@@ -1417,21 +1417,74 @@ func (w *WalletKit) fundPsbtInternalWallet(account string,
 				return err
 			}
 
+			eligible := make([]*lnwallet.Utxo, 0, len(utxos))
+			for _, utxo := range utxos {
+				// Don't use inputs which are unconfirmed and
+				// used by the sweeper.
+				if utxo.Confirmations == 0 &&
+					w.cfg.Sweeper.IsSweeperOutpoint(
+						utxo.OutPoint,
+					) {
+
+					log.Warnf("Cannot use unconfirmed "+
+						"utxo=%v because it is "+
+						"unstable and could be "+
+						"replaced", utxo.OutPoint)
+
+					continue
+				}
+				eligible = append(eligible, utxo)
+			}
```

Comment:
Good time to use [`fn.Filter`](https://github.com/lightningnetwork/lnd/blob/b1175514f9854b965d4716ac015ea15387c63081/fn/slice.go#L39-L51)